Dockerfile

```diff
@@ -5,24 +5,24 @@ ENV GOOS=linux
 
 RUN apk add --no-cache make git
 
-WORKDIR /go/src/github.com/netlify/gotrue
+WORKDIR /go/src/github.com/supabase/gotrue
 
 # Pulling dependencies
 COPY ./Makefile ./go.* ./
 RUN make deps
 
 # Building stuff
-COPY . /go/src/github.com/netlify/gotrue
+COPY . /go/src/github.com/supabase/gotrue
 RUN make build
 
 FROM alpine:3.15
```

Comment:
Should probably bump this too - opened a [PR](https://github.com/supabase/gotrue/pull/883)